web.test.tests/web.test.tests/Pages/HistoryPage.cs

```diff
@@ -0,0 +1,121 @@
+﻿using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace web.test.tests.Pages
+{
+    class HistoryPage
+    {
+        private IWebDriver driver;
+
+        public static String ClearBtnId = "clear";
+
+        public HistoryPage(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        public IWebElement Calculator => driver.FindElement(By.XPath("//div[text()='Calculator']"));
+        public IWebElement ClearBtn => driver.FindElement(By.Id(ClearBtnId));
+
+        public HistoryPage Clear()
+        {
+            ClearBtn.Submit();
+            return new HistoryPage(driver);
```

Comment:
If you need to return the same page, you may use 'this'.

```suggestion
            return this;
```

Reply:
Done